tasks/create_http_task_with_token.py

```diff
@@ -40,6 +40,7 @@ def create_http_task(project,
     # location = 'us-central1'
     # url = 'https://example.com/task_handler'
     # payload = 'hello'
+    # service_account_email = 'service-account@my-project-id.iam.gserviceaccount.com';
```

Comment:
nit: Could this be placed between `url` and `payload` to keep the same order as the function args?